cwms-data-api/src/main/java/cwms/cda/data/dao/BlobDao.java

```diff
@@ -158,14 +158,14 @@ public void update(Blob blob, boolean ignoreNulls) {
             throw new NotFoundException("Null blob provided to update");
         }
 
-        if (!blobExists(blob.getOfficeId(), blob.getId())) {
-            throw new NotFoundException("Unable to find blob with id " + blob.getId() + " in office " + blob.getOfficeId());
+        if (!blobExists(blob.getOfficeId(), blob.getId().toUpperCase())) {
+            throw new NotFoundException("Unable to find blob with id " + blob.getId().toUpperCase() + " in office " + blob.getOfficeId());
         }
 
         dsl.connection(c -> CWMS_TEXT_PACKAGE.call_STORE_BINARY(
                 getDslContext(c, blob.getOfficeId()).configuration(),
                 blob.getValue(),
-                blob.getId(),
+                blob.getId().toUpperCase(),
```

Comment:
while all the comparisons should be upper case, we do want the user to be able to set the casing to whatever they want for visualization and readability.